yarn-project/noir-libs/easy-private-state/src/easy_private_state.nr

```diff
@@ -94,10 +94,10 @@ impl EasyPrivateUint {
 
         let owner_key = get_public_key(owner);
 
-        let _context = self.context.unwrap();
+        let maybeContext = self.context.unwrap();
         emit_encrypted_log(
-            _context,
-            (*_context).this_address(),
+            maybeContext,
+            (*maybeContext).this_address(),
```

Comment:
Ouch, I think I explained myself wrong on the other PR, sorry. I meant that the property in the struct (ie what's today `self.context`) should be `self.maybe_context` (I also screwed up the casing, we're using snake case in Noir, right?). Rationale is: an option type is ["maybe"](https://wiki.haskell.org/Maybe) something, but when you unwrap it, you have the certainty that it **is** an instance of that something (or you have blown up).

Also, is it possible this happened in more than a single file?

Reply:
I should try using my brain when reading your comments. Addressed in [e47fed5](https://github.com/AztecProtocol/aztec-packages/pull/1976/commits/e47fed53baaf564247451f2cf3f4b1abcb2d0c9b). Thanks